dallo/src/lib.rs

```diff
@@ -26,7 +26,7 @@ mod types;
 pub use types::*;
 
 /// How many bytes to use for scratch space when serializing
-pub const SCRATCH_BUF_BYTES: usize = 16;
+pub const SCRATCH_BUF_BYTES: usize = 64;
```

Comment:
Why the increase in scratch buffer size?

Reply:
It's still a number pulled at random, still in need of investigation, just bumped it to have some more headroom.